config/initializers/services.rb

```diff
@@ -1,19 +1,30 @@
 # frozen_string_literal: true
 
-if Chipmunk.config.checkpoint&.database&.url
-  Checkpoint::DB.config.url = Chipmunk.config.checkpoint.database.url
-end
+require "chipmunk_bag"
 
-if Chipmunk.config.keycard&.database&.url
-  Keycard::DB.config.url = Chipmunk.config.keycard.database.url
+def assign_db(lhs, rhs)
+  if rhs.is_a? String
+    lhs.url = rhs
+  elsif rhs.respond_to?(:has_key?)
+    if rhs["url"]
+      lhs.url = rhs["url"]
+    else
+      lhs.opts = rhs
+    end
+  end
```

Comment:
Are we actually using a string anywhere? My impression is that it should be hash-only.

Reply:
We're not. This was for an intermediate solution, and seemed okay to leave around.